### PR TITLE
added improved MTBLS1 workflow

### PR DESCRIPTION
**Pull request by korseby**
Hi @pcm32 

Here is the improved version of the NMR MTBLS1 workflow. I added your suggestions + pull request you sent to @c-ruttkies repo.

I also added one really great galaxy module thanks to @djacob65. It produces a fancy stacked plot out of the data matrix. I also updated the workflow.

Please merge. I'm now officially on vacation. ;)

### CODE REVIEW

**Comment by pcm32** [comment]
Will do today @rsalek, but I'll need you to test it.

File: Dockerfile [hunk 1]
```diff
@@ -45,6 +45,8 @@ RUN chmod u+x /galaxy/ansible/run_galaxy_config.sh
 # Missing XCMS datatypes for w4m
 COPY external-datatypes/rdata_xcms_datatype.py /galaxy/lib/galaxy/datatypes/
 COPY external-datatypes/rdata_camera_datatype.py /galaxy/lib/galaxy/datatypes/
+COPY external-datatypes/no_unzip_datatypes.py /galaxy/lib/galaxy/datatypes/
```
Comment (pcm32) [inline Dockerfile]: We should avoid this in the future, but will merge as of now.

**Comment by djacob65** [comment]
Hi all,

I made modification in that way within the Rnmr1D module.
I replaced the  '-z, --zip' option by a new one '-i, --input' which accepts two types of input :  either the raw spectra directory (as wished) or the ZIP file of raw spectra (as previous)

I've just 'pushed' the new changes in the Rnmr1D module (https://bitbucket.org/nmrprocflow/rnmr1d). 
See a detailled example at https://bitbucket.org/nmrprocflow/rnmr1d/src/d5216a69c828/examples/

Cheers

**Comment by rsalek** [comment]
Fantastic :)

**Comment by rsalek** [comment]
Hi @korseby can we update this? So I can give it a try without the "zip" step :)

**Comment by korseby** [comment]
Fantastic news while I was away! Of course, this means that there is no "-z" parameter any longer? The "-i" parameter should make it fairly easy for us.

The rnmr1d galaxy module has to have `multiple="True"` set. And we have to implement a rule that the galaxy_xxx.dat are renamed, but we can take the rules from the zip module.

Let's talk about this in the hangout in a few hours. Can you also make sure that when you update the galaxy modules, you also push changes to galaxy dir in the repos of the tools (no pull request needed)? Because this is where all of our development work happens.

**Comment by pcm32** [comment]
Gentlemen, this is a merged pull request thread. I suggest we move this discussions to an issue were we track the further improvements needed:

https://github.com/phnmnl/phenomenal-h2020/issues/8

@korseby I did push all changes to c-ruttkies development from where you did the pull request (that is why my commits got incorporated). I think that Reza was asking for the ability to specify the MTBLS ID on the initial query, I have already added that as part of this pull request (so you don't need to do it).

**Pull request merged**